### DISCUSSION TITLE
Editable properties or as an option ?

### DISCUSSION THREAD

**Discussion by bouhpwet** [Xanashi/Icaros#78] [Feature Requests]
Would it be possible to have the _Details_  tab of a video **editable** ? Unless mistaken, all details become read-only after installing Icaros. For instance, when looking at the _Details_  tab of an *.MP4 video, I can't change the Title or the Comments.

Alternatively, it would be great if the property handler would be an optional install.

Many thanks for your consideration.

**Comment by Xanashi** [comment]
Hi bouhpwet,

You can enable/disable the property handler via IcarosConfig, you can even just disable MP4 property handling and keep all other formats activated. MP4 property handling is actually disabled by default in Icaros for this exact reason.
Editable properties in Icaros is further down the line.

**Reply by bouhpwet** [reply]
Thank you for the quick and clear reply.
The thing that puzzles me is: After resetting the property filetypes, the details tab are indeed back to default, but I cannot edit the metadata for format supported by Icaros handler (i.e. AVI, MP4...) but I still can do it for other formats (ASF, WMV...).
Any idea or suggestion ?

**Reply by Xanashi** [reply]
You should be able to edit MP4 again, as soon as it is disabled/unchecked in Icaros. 
I'm not sure AVI is editable in Windows. I believe that requires a thirdparty software. 
If MP4 doesn't work, you could try and restart Explorer, and you can do that on the Tools page in IcarosConfig.
